client/gutenberg/extensions/map/view.js

```diff
@@ -13,8 +13,8 @@ import apiFetch from '@wordpress/api-fetch';
 window &&
 	window.addEventListener( 'load', function() {
 		const frontendManagement = new FrontendManagement();
-		const url = '/?rest_route=/jetpack/v4/service-api-keys/mapbox';
-		apiFetch( { url, method: 'GET' } ).then( result => {
+		const path = '/jetpack/v4/service-api-keys/mapbox';
+		apiFetch( { path, method: 'GET' } ).then( result => {
```

Comment:
`method: 'GET'` seems redundant, I think the underlying [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch) defaults to `GET` already.

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/28772/commits/ab3e8bb86805cf829af348d00f5ed1df1923288a